### PR TITLE
Update SeLion document with mobile platform, common interface for and…

### PR DESCRIPTION
**Pull request by alizelzele**
Documentation for #265 (common interface for android and ios)

### CODE REVIEW

File: html/mobileNativePageAppAutomation.html [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+         <em>src/main/resources/GUIData/</em> with name <strong>NativeAppTestPage.yaml</strong>. 
+         It should have the following content:
+      <pre data-src="../sample-files/yamlpage-mobilenativeapp.yaml" class="language-markup"></pre>
+    </li>
+    <li>First the <code>@Test</code> annotated method needs to also be annotated with <code>@MobileTest</code>.
+      This indicates to SeLion that the <code>@Test</code> code will be performed against a mobile device or emulator.</li>
+    <li>A typical test would look like this:
+      <pre data-src="../sample-files/mobilenativeapp-pageobject.java"></pre>
+    </li>
+    <li>Since usually we use mobile platform for both android and ios it is a good idea to config every thing from test suite.
```
Comment (mach6) [inline html/mobileNativePageAppAutomation.html]: `everything`, one word

Comment (mach6) [inline html/mobileNativePageAppAutomation.html]: should be `from the test suite`

File: html/mobileNativePageAppAutomation.html [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    parameters that were defined in @MobileTest should be defined in test suite, plus we need to define the platform to use.
+    </li>
+    <li> a sample test suite for mobile platform test loks like this:
```
Comment (mach6) [inline html/mobileNativePageAppAutomation.html]: should be `looks like this`

Comment (mach6) [inline html/mobileNativePageAppAutomation.html]: for `a` mobile platform test

**Comment by mach6** [comment]
Thanks for seeding this change @alizelzele 

All-in-all, I feel this PR is going to take a bit more work. I feel having three sections (1) for mobile app, (2) for android, (3) for iOS is going to be very confusing to a newcomer. Perhaps we should revise, edit, and/or remove existing documentation to support this change rather than adding entire new sections.

**Comment by alizelzele** [comment]
@mach6 I corrected typo and grammatical problems, will revise my commit to create better docs.

**Comment by alizelzele** [comment]
@mach6 Sorry for late response. I updated the docs, can you please take a look at it.

**Comment by sebady** [comment]
merged

**Pull request closed without merging**